classes/invitation/invitations/userRoleAssignment/resources/BaseUserRoleAssignmentInviteResource.php

```diff
@@ -51,8 +51,8 @@ protected function transformUserGroups(?array $userGroups)
                 'userGroupId' => $userGroup['userGroupId'],
                 'userGroupName' => $userGroupModel->getLocalizedData('name'),
                 'masthead' => $userGroup['masthead'],
-                'dateStart' => $userGroup['dateStart'],
-                'dateEnd' => $userGroup['dateEnd'],
+                'dateStart' => $data['dateStart'] ?? null,
```

Comment:
Should this and the line below still use `$userGroup`, rather than `$data`?

Reply:
Yes! Fixed